critiquebrainz/db/review.py

```diff
@@ -83,6 +84,7 @@ def get_by_id(review_id):
                    review.language,
                    review.source,
                    review.source_url,
+                   review.publish_time,
```

Comment:
Since we are adding another column to the table (for publish date), it is important to remove the subquery for creating `created_time` and the corresponding entry of `created_time.created` in `SELECT`, as that part is now unnecessary.

Reply:
Also, for adding the `Last Revised: ` information, you can make use of the `revision.timestamp`, so I would suggest it to rename the column as `last_revision_timestamp`.